objects.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import "github.com/faiface/pixel"
+
+// Gravity is vector representing standard gravity force
+var Gravity = pixel.Vec{
+	X: 0.0,
```

Comment:
Why this is uppercased?

Reply:
Because it is an exported variable and those should be uppercased in `Go`